src/autoqasm/operators/arithmetics.py

```diff
@@ -0,0 +1,47 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+"""Operators for arithmetic operators: // """
+
+from __future__ import annotations
+
+from autoqasm import program
+from autoqasm import types as aq_types
+
+from .utils import _register_and_convert_parameters
+
+def fd_(a: int, b: int) -> int | aq_types.IntVar:
```

Comment:
you'll also want to add your new operator to the list of operators `__init__.py`:
https://github.com/amazon-braket/autoqasm/blob/main/src/autoqasm/operators/__init__.py